src/promptflow/tests/sdk_cli_azure_test/conftest.py

```diff
@@ -12,14 +12,9 @@
 
```

Comment:
let's add comments somewhere to highlight we don't have annotation in azure tests, in case others might introduce this again.

Reply:
maybe in pr description, and sync this in chat group - I suppose only SDK team will touch SDK Azure part